sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -3416,18 +3428,31 @@ pi_result cuda_piextUSMDeviceAlloc(void **result_ptr, pi_context context,
                                    pi_device device,
                                    pi_usm_mem_properties *properties,
                                    size_t size, pi_uint32 alignment) {
+  // from empirical testing with CUDA 10.2 on a Tesla K40
+  static constexpr pi_uint32 max_alignment = 0x200;
```

Comment:
Should we make the constant global to share it between the different functions?

Reply:
I was considering if host and device could have different default alignments - but given the lack of details, I guess we could use a global `static` for the moment.

Reply:
That is a valid thought!